nebula/ui/components/VPNFlickable.qml

```diff
@@ -106,7 +106,7 @@ Flickable {
         minimumSize: 0
 
         opacity: hideScollBarOnStackTransition && (vpnFlickable.StackView.status !== StackView.Active) ? 0 : 1
-        visible: !windowHeightExceedsContentHeight
```

Comment:
The old logic here basically says only show scroll bars when `flickable.contentHeight > window.height`. This logic for showing scroll bars didn't make a whole lot of sense to me. What if the flickable's viewport is only 100px tall, and needs to show 200px of content? 200px is surely going to be smaller than the window height, but a scroll bar should still be shown because it is scrollable.

We seem to typically base `flickable.interactive` on `flickable.contentHeight > flickable.height`, meaning, if all the content can't be shown in the viewport, allow scrolling. We currently always show scrolls bars when scrolling is available, so this remains consistent with what already exists